FuncGameDev/FSharp/Unity/Update.fs

```diff
@@ -36,14 +42,43 @@ type Updater() =
     member this.checkGameOver() =
```

Comment:
This function is very large and ugly. I would recommend pulling the "score update" part of it out into another function and calling it twice (once in each case) instead.

Reply:
As for where to put this hypothetical new function: the score saving service seems like an excellent location. It has no business being in the update loop anyhow, frankly.